[2022-09-05 03:07:54,903] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-29T00:00:00+00:00 [queued]>
[2022-09-05 03:07:55,050] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-29T00:00:00+00:00 [queued]>
[2022-09-05 03:07:55,051] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:55,052] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:55,061] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:55,157] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-08-29T00:00:00+00:00
[2022-09-05 03:07:55,205] {standard_task_runner.py:52} INFO - Started process 457 to run task
[2022-09-05 03:07:55,246] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-08-29T00:00:00+00:00', '--job-id', '94', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpfrylgn78', '--error-file', '/tmp/tmp2zms60qf']
[2022-09-05 03:07:55,251] {standard_task_runner.py:77} INFO - Job 94: Subtask second_task
[2022-09-05 03:07:55,596] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-08-29T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:56,175] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-29T00:00:00+00:00
[2022-09-05 03:07:56,177] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:56,182] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:07:56,223] {bash.py:169} INFO - Output:
[2022-09-05 03:07:56,227] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:07:56,237] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:56,554] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220829T000000, start_date=20220905T030754, end_date=20220905T030756
[2022-09-05 03:07:56,770] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:56,819] {local_task_job.py:146} INFO - Task exited with return code 0
